src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -138,7 +138,17 @@ public void On(string methodName, Type[] parameterTypes, Func<object[], Task> ha
 
         private async Task<ReadableChannel<object>> StreamAsyncCore(string methodName, Type returnType, object[] args, CancellationToken cancellationToken)
         {
-            var irq = InvocationRequest.Stream(cancellationToken, returnType, GetNextId(), _loggerFactory, out var channel);
+            var irq = InvocationRequest.Stream(CancellationToken.None, returnType, GetNextId(), _loggerFactory, out var channel);
+            if (cancellationToken.CanBeCanceled)
```

Comment:
See, the fact that we're plumbing it through as a `CancellationToken` even when streaming indicates to me that we should consider supporting it in regular invocations. We can talk about it after this PR, but just pointing out that these are basically the same.